tests/hypothesis_strategies/kernels.py

```diff
@@ -19,6 +20,8 @@
 from ..hypothesis_strategies.basic import finite_floats
 from ..hypothesis_strategies.priors import priors
 
+_positive_finite_float = partial(finite_floats, min_value=0.0, exclude_min=True)
```

Comment:
looks like this might be useful to put into `..hypothesis.strategies.basic`, no?

Reply:
Agree, it should at least be treated the same way as `finite_floats`, so in particular it feels weird to have them in different places and having one private and one public.

Reply:
done